src/functionalTest/java/uk/gov/hmcts/reform/draftstore/data/DraftStoreDAOTest.java

```diff
@@ -9,14 +9,13 @@
 import org.springframework.test.context.ActiveProfiles;
 import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
 import org.springframework.transaction.annotation.Transactional;
-import uk.gov.hmcts.reform.draftstore.domain.Draft;
```

Comment:
General question on this PR:
What happens if the client sends a secret and the document isn't encrypted?
During the transition stage you wouldn't know if its encrypted or not

Reply:
we don't make any changes to it and return plain text. On next update it will be encrypted though and plaintext db column will be set to null.

See `FromDbModelMapper` for the read part

Reply:
```
// only during transition stage
String documentToReturn =
    dbDraft.encryptedDocument == null
        ? dbDraft.document
        : CryptoService.decrypt(dbDraft.encryptedDocument, secret);
```

Reply:
also, notice that this happens 'per draft', so it is possible to read a mix of old (plaintext) and new (encrypted) drafts from db for given user.